Python/bytecodes.c

```diff
@@ -1550,6 +1548,7 @@ dummy_func(
                 }
                 Py_INCREF(value);
             }
+            Py_DECREF(class_dict);
```

Comment:
This DECREF won't happen if we GOTO_ERROR on like 1547.

Reply:
Yes, this is the expected behavior. I'll add some notes on the first comment.